install.sh

```diff
@@ -94,133 +103,128 @@ esac
 
 # functions to maintain packages
 function download_check () {
-    cd $CREW_BREW_DIR
+    cd "${CREW_BREW_DIR}"
 
     #download
-    echo "Downloading $1..."
-    curl --progress-bar -C - -# -L --ssl $2 -o "$3"
```

Comment:
Yeah, this random `#` showing up in the output was annoying. :+1:

Reply:
Yeah, the `--progress-bar` long option is much better than `-#`.